seed/network-connection.sh

```diff
@@ -90,6 +90,69 @@ function configure_tcp() {
 
 configure_tcp
 
+# for each cidr config, it looks first at its env var, then a local file (which may be a volume mount), then the default
+baseConfigDir="/init-config"
+fileServiceNetwork=
+filePodNetwork=
+fileNodeNetwork=
+[ -e "${baseConfigDir}/serviceNetwork" ] && fileServiceNetwork=$(cat ${baseConfigDir}/serviceNetwork)
```

Comment:
maybe use`[[` instead of `[`

Reply:
don't know if we really need the bash extension here

Reply:
the function is the same, it would just save the quotes